tools/domspurge/purge.py

```diff
@@ -0,0 +1,344 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+import argparse
+import json
+import logging
+from datetime import datetime
+from typing import Tuple, List
+
+from cassandra.auth import PlainTextAuthProvider
+from cassandra.cluster import Cluster, NoHostAvailable, ExecutionProfile, EXEC_PROFILE_DEFAULT
+from cassandra.policies import RoundRobinPolicy, TokenAwarePolicy
+from dateutil import parser as du_parser
+from dateutil.relativedelta import relativedelta
+from six.moves import input
+from tqdm import tqdm
+
+logging.getLogger('webservice.NexusHandler').setLevel(logging.CRITICAL)
+
+from webservice.algorithms.doms.DomsInitialization import DomsInitializer
+
+
+logging.basicConfig(level=logging.INFO)
```

Comment:
Maybe log to stdout as well since this is a script?

Reply:
Done